README.md

```diff
@@ -35,7 +35,9 @@ Python 3.7
 	-   SOUNDCLOUD_CLIENT_ID - API key for SoundCloud API
 
 -   Optional environment variables:
-    	-   BOT_ADMINS_CHAT - if you set admins chat (eg @our_chat) bot messages will have 'report button'. if click - bot forwards messages to admins chat
+    	
+	-   BOT_ADMINS_CHAT - if you set admins chat (eg @our_chat) bot messages will have 'report button'. if click - bot forwards messages to admins chat
+	-    EXAMPLE_MEDIA - if you example photo/gif URL it will be shown on /start and /help user commands
```

Comment:
`-    EXAMPLE_MEDIA` - looks like too many spaces

Reply:
it's ok 
![image](https://user-images.githubusercontent.com/45029602/56031442-2c9be500-5d28-11e9-9bd7-7c390ce266ce.png)